<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Example1Class</class>
 <widget class="QMainWindow" name="Example1Class">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Example1</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*[mandatoryField=&quot;true&quot;]
{
	background-color: yellow;
}

QCheckBox::indicator,QRadioButton::indicator{
 width:20px;
 height:20px;
}

QCheckBox,QRadioButton{
	spacing: 8px;
}

QCheckBox:checked,QCheckBox:hover{
	color: white;
}

QCheckBox:!Checked{
	color: aqua;
}

QLineEdit{
	padding: 0px 15px 0px 0px;
}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QCheckBox" name="checkBox">
      <property name="text">
       <string>CheckBox</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QComboBox" name="genderComboBox">
      <item>
       <property name="text">
        <string>Male</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Female</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string/>
       </property>
      </item>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="nameLineEdit">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
     </widget>
    </item>
    <item row="0" column="4">
     <widget class="QSpinBox" name="ageSpinBox"/>
    </item>
    <item row="0" column="3">
     <widget class="QRadioButton" name="radioButton">
      <property name="text">
       <string>RadioButton</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="3">
     <widget class="QListView" name="listView"/>
    </item>
    <item row="1" column="3">
     <widget class="QPushButton" name="btnOK">
      <property name="styleSheet">
       <string notr="true">QPushButton[text=&quot;OK&quot;]{
	color: green;
}</string>
      </property>
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
    </item>
    <item row="1" column="4">
     <widget class="QPushButton" name="btnCancel">
      <property name="styleSheet">
       <string notr="true">QPushButton[text=&quot;Cancel&quot;]{
	color: red;
}</string>
      </property>
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Example1.qrc"/>
 </resources>
 <connections/>
</ui>
